lib/CommandHandler.js

```diff
@@ -42,14 +42,19 @@ class CommandHandler {
             return logger.debug(`Command '${commandName}' was not found.`);
         }
 
+        if(!message.channel.permissionsOf(this.client.client.user.id).has("sendMessages")) {
+            addTemporaryReaction(message, "📝");
+            return logger.debug(`Cannot send messages in ${message.channel.name}`);
```

Comment:
Should be channel ID, or both. If both, then the cooldown debug message needs to be updated as well.